src/gluonts/dataset/repository/_uber.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+from pathlib import Path
+from typing import Optional
+
+import pandas as pd
+
+from gluonts.dataset.repository._util import metadata
+
+
+def generate_uber_dataset(
+    dataset_path: Path, uber_freq: str, prediction_length: Optional[int] = None
+):
+    from gluonts.dataset.repository.datasets import default_dataset_path
+
+    uber_csv_path = default_dataset_path / "uber-raw-data-janjune-15.csv"
+    if not os.path.exists(uber_csv_path):
+        raise RuntimeError(
+            "The uber data is available at "
+            "https://github.com/fivethirtyeight/uber-tlc-foil-response/blob/"

```

Comment:
Can you check if we automatically download other datasets? This would make things much easier for users.

Reply:
The csv file is very large and we don't have access to the raw data and can only download a zip file. I will try to add a function to automatically download a zip file and read the raw data.

Reply:
`pd.read_csv` might be able to handle it for us already.